beetsplug/alternatives.py

```diff
@@ -117,6 +117,12 @@ def parse_config(self, config):
             dir = os.path.join(self.lib.directory, dir)
         self.directory = dir
 
+        if not config['linktype'] in ['absolute', 'relative']:
+            # TODO: Raising some kind of exception
+            config['linktype'] = 'absolute'
+
+        self.relativelinks = bool(config['linktype'] == 'absolute')
```

Comment:
We initialize `self.relativelinks` here but only use it in in the `SymlinkView` sub class. This means the code here is not necessary for the `External` class to function. It might be better to move this code into `SymlinkView.parse_config` to clearly separate concerns. This would also consolidate all the code that touches `config['linktype']` in one place.

Also I don’t think wrapping this in `bool()` is necessary

Reply:
Moved in next commit!